php/Terminus/Commands/WorkflowsCommand.php

```diff
@@ -73,7 +75,54 @@ public function show($args, $assoc_args) {
         $this->log()->info('Workflow has no operations');
       }
     } else {
-      $this->output()->outputRecordList($workflow_data);
+      $this->output()->outputRecord($workflow_data);
+    }
+  }
+
+  /**
+   * Show quicksilver logs from a workflow
+   *
+   * ## OPTIONS
+   * [--workflow_id]
+   * : Uuid of workflow to fetch logs for
+   * [--site=<site>]
+   * : Site from which to list workflows
+   *
+   * @subcommand logs
+   */
+  public function logs($args, $assoc_args) {
```

Comment:
The name of this subcommand is inconsistent with the pattern we've been trying to establish, that is that the subcommand or action must be a verb.